### PR TITLE
Add display plugins by topic and refactor main application

### PR DESCRIPTION
**Pull request by Sarath18**

### CODE REVIEW

File: ign_rviz/include/ignition/rviz/rviz.hpp [hunk 1]
```diff
@@ -24,277 +24,171 @@
...
+    TypeRole
+  };
+
+  // Constructor
+  explicit TopicModel(QObject * _parent = 0);
+
+  /**
+   * @brief Add a topic to list view
+   * @param[in] _name Topic name
+   */
```
Comment (ahcorde) [inline ign_rviz/include/ignition/rviz/rviz.hpp:64]:
```suggestion
   * @param[in] _msgType
   */
```

**Review by ahcorde — CHANGES_REQUESTED** [review]
Can edit the title to include the changes of the rviz.hpp class ?

**Comment by Sarath18** [comment]
**Add display plugins by topic and refactor main application**
Is this okay?

**Comment by ahcorde** [comment]
Sure :+1:

**Review by ahcorde — APPROVED** [review]

**Pull request merged**